src/NHibernate/ISession.cs

```diff
@@ -763,14 +771,9 @@ public partial interface ISession : IDisposable
 		/// Get the current Unit of Work and return the associated <c>ITransaction</c> object.
 		/// </summary>
 		// Since v5.3
-		[Obsolete("Use CurrentTransaction instead, and check for nulls.")]
+		[Obsolete("Use GetCurrentTransaction extension method instead, and check for nulls.")]
```

Comment:
Why `nulls`. Shouldn't it be simply `null`? Typo?

Reply:
Just a kind of contraction of `null values`, not really good language anyway. Will adjust.